ui/hooks/useTokenFiatAmount.js

```diff
@@ -33,8 +34,12 @@ export function useTokenFiatAmount(
   const currentCurrency = useSelector(getCurrentCurrency);
   const userPrefersShownFiat = useSelector(getShouldShowFiat);
   const showFiat = overrides.showFiat ?? userPrefersShownFiat;
+  const contractExchangeTokenKey =
+    Object.keys(contractExchangeRates).find((i) =>
+      isEqualCaseInsensitive(i, tokenAddress),
+    ) || '';
   const tokenExchangeRate =
-    overrides.exchangeRate ?? contractExchangeRates[tokenAddress];
+    overrides.exchangeRate ?? contractExchangeRates[contractExchangeTokenKey];
```

Comment:
Instead of representing "not found" with `contractExchangeTokenKey['']`, would it be more accurate to say this? Also would it be more clear to represent the key as `key` rather than `i`?

``` suggestion
  const contractExchangeTokenKey =
    Object.keys(contractExchangeRates).find((key) =>
      isEqualCaseInsensitive(key, tokenAddress),
    );

  const tokenExchangeRate =
    overrides.exchangeRate ?? (contractExchangeTokenKey && 
  contractExchangeRates[contractExchangeTokenKey]);

Reply:
Yeah that makes sense!